frame/election-provider-support/solution-type/Cargo.toml

```
@@ -1,5 +1,5 @@
 [package]
-name = "sp-npos-elections-solution-type"
+name = "frame-npos-elections-solution-type"
```

```suggestion
name = "frame-election-provider-solution-type"
```

This is nested in `election-provider-support`, so ideally we should call it as such.